### DISCUSSION TITLE
Backpack shall drop itself when destroyed in Creative Mode

### DISCUSSION THREAD

**Discussion by xFirefalconx** [P3pp3rF1y/SophisticatedBackpacks#643] [Ideas]
Imagine accidentally mistaking creative mode for survival mode, trying to destroy a backpack to drop it for an upgrade and irreversible destroying it, including its contents. 
Spoiler alert that just happened to me :') 

So my idea: Make a config or even a default to make it drop itself when destroyed in Creative Mode. Maybe check for contents and if there are none (including if there are no upgrades in the upgrade slots), destroy it, instead. 

I bet I am not the only person who sometimes got these brain lags and does dumb things so this might be a _sophisticated_ solution for those oopsies. 

Thanks for reading.

P.S.: Hope this is possible for 1.18.2.

**Comment by awesomebossdj7** [comment]
I say it shouldn't even be a config, it should be a feature already, shulker boxes do this in creative mode when they have items inside. 


also yeah it should be possible

**Comment by Twisted-Code** [comment]
As a workaround, there is a command to restore lost/destroyed backpacks. I'm not sure if it's in 1.18.2, but it's definitely in 1.19 builds
